File: Hulk.java
package round2;

import java.util.Scanner;

public class Hulk {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int n = scanner.nextInt();
		if (n == 1) {
			System.out.println("I hate it\r\n"
					+ "");
		} else if (n == 2) {
			System.out.println("I hate that I love it\r\n"
					+ "");
		} else if (n == 3) {
			System.out.println("I hate that I love that I hate it\r\n"
					+ "");
		}
	}
}

File: hulk.cpp
#include <bits/stdc++.h>
using namespace std;
int n;
int main()
{
    cin >> n;
    // string m1[1000] = {"I hate it", "I love it", "I hate it"};
    for (int i = 0; i < n; i++)
    {
        if (i % 2 == 0)
        {
            cout << "I hate ";
        }
        else
        {
            cout << "I love ";
        }
        if (i != n - 1)
        {
            cout << "that ";
        }
        else
        {
            cout << "it ";
        }
    }
    return 0;
}